easybuild/easyblocks/generic/rpm.py

```diff
@@ -46,6 +48,41 @@
 from easybuild.tools.run import run_cmd
 
 
+_log = fancylogger.getLogger('easyblocks.generic.rpm')
+
+
+def rebuild_rpm(rpm_path, targetdir):
+    """Rebuild the RPM on the specified location, to make it relocatable."""
+    rpmmacros = os.path.join(expanduser('~'), '.rpmmacros')
+    if os.path.exists(rpmmacros):
+        _log.error("rpmmacros file %s found which will override any other settings, so exiting." % rpmmacros)
+
+    rpmrebuild_tmpdir = os.path.join(tempfile.gettempdir(), "rpmrebuild")
+    env.setvar("RPMREBUILD_TMPDIR", rpmrebuild_tmpdir)
+
+    try:
+        if not os.path.exists(rpmrebuild_tmpdir):
+            os.makedirs(rpmrebuild_tmpdir)
+            _log.debug("Created RPMREBUILD_TMPDIR dir %s" % rpmrebuild_tmpdir)
+        if not os.path.exists(targetdir):
+            os.makedirs(targetdir)
+            _log.debug("Created target directory for rebuilt RPMs %s" % targetdir)
+    except OSError, err:
+        _log.error("Failed to create directories for rebuilding RPM: %s" % err)
+
+    _log.debug("Rebuilding %s in %s to make it relocatable" % (rpm_path, targetdir))
+    cmd = ' '.join([
+        "rpmrebuild -v",
+        r"""--change-spec-whole='sed -e "s/^Prefix:.*/Prefix:    \//"'""",
+        r"""--change-spec-whole='sed -e "s/^\(.*:[ ]\+\..*\)/#ERROR \1/"'""",
```

Comment:
why not `Prefix: \/` instead of `Prefix:      \/`?

I really don't like it if we don't know what the second part does. As far as I can see, it replaces everything that start with a `.`?